project-resources/PROJECT_FUNDS.md

```diff
@@ -0,0 +1,24 @@
+# OpenJS Foundation Project-specific Initiative Funds
+
+The OpenJS Foundation provides a set of monetary funds for projects of the OpenJS Foundation. 
+These funds are designed to support collaboration and incentive project-specific initiatives that have high-impact within the individual project's scope.
+
+Alongside these lines, the fund exist to support the development of these 
+initiatives and give visibility/recognition to the champions of these project-specific initiatives, highligthing the individuals behind them.
+
+## F.A.Q.
+
+#### How might a project request for funds?
+
+Funds should be requested on an ad-hoc basis through a PR requesting the funds. The requests do not need to follow a specific template,
```

Comment:
FWIW, I think offering a template makes everyone's life easier, so I think we just should do that.

Reply:
Makes sense. WDYT the template should contain?

Reply:
Which points, the ones @joesepi wrote?

Reply:
My understanding is @erickzhao was referring those in the quoted sentence:

> To access the funds, the project would need to prepare a 1-pager plan, including the purpose and type fo activity, success criteria, timeline, and funds requested

Reply:
Oh my goodness, how I missed that! Sorry @erickzhao 🙈

Reply:
There seemed to be agreement to have a template for this. Can we bake it into this PR?

Reply:
Sure thing!